### DISCUSSION TITLE
Multi User Git

### DISCUSSION THREAD

**Discussion by skumaravelan** [apache/camel-karavan#980] [General]
Karavan install uses git credentials to access the repo for storing project files, when other persons adds project and how we could audit the user check-in details?
Is there any way to enable multi user Karavan UI?

**Comment by jwgmeligmeyling** [comment]
Multi user Karavan is possible when using OAuth as security provider, however it does not use user specific git credentials as of yet.

**Reply by skumaravelan** [reply]
How could we use OAuth as security provider ?

**Reply by TheHandOfNOD** [reply]
There is a installation parameter called --auth which takes 3 possible values: public, basic, oidc.

**Comment by TheHandOfNOD** [comment]
For me the question is why it's currently only possible to push via access token of 1 single git account? There will be almost not a single project where only 1 developer will use Karavan for himself, but there will always be a team pushing features into several branches...
